data/json/effects.json

```diff
@@ -56,7 +56,7 @@
         "type": "effect_type",
         "id": "asked_to_train",
         "name": ["Asked to Train"],
-        "desc": ["AI tag for asking to NPCs to train you.  This is a bug if you have it."]
+        "desc": ["You've recently asked this NPC to train you. You'll need to wait a while till you can ask again."]
```

Comment:
Missing double space after `train you.`

Also suggest [changing till](https://www.merriam-webster.com/words-at-play/should-you-use-until-or-till-or-til) and [a while](https://www.merriam-webster.com/words-at-play/awhile-usage). May be better to just be more specific in the second sentence (since a while is meant to be a short period of time, and the effect lasts six hours?):

`You'll need to wait several hours before [they'll/this NPC will] consider training you again.`